api/python/cellxgene_census/tests/test_get_anndata.py

```diff
@@ -197,7 +195,7 @@ def test_get_anndata_obs_embeddings(lts_census: soma.Collection, obs_embeddings:
         obs_embeddings=obs_embeddings,
     )
 
-    assert len(ad.obsm.keys()) == 3
+    assert len(ad.obsm.keys()) == 2
```

Comment:
What is the behavior if we generate multiple scvi embeddings for a release? And do we plan to add embeddings to old LTS releases?

I'm wondering if this test should just run on the older release.

Reply:
We don't really support having multiple embeddings per release. In this case, we would yank the previous version and replace it with the new one.